services/core-web/src/components/admin/UpdateMinespaceUser.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import { getMineNames } from "@common/selectors/mineSelectors";
+import { fetchMineNameList } from "@common/actionCreators/mineActionCreator";
+
+import CustomPropTypes from "@/customPropTypes";
+import EditMinespaceUser from "@/components/Forms/EditMinespaceUser";
+
+const propTypes = {
+  fetchMineNameList: PropTypes.func.isRequired,
+  updateMinespaceUser: PropTypes.func.isRequired,
+  mines: PropTypes.arrayOf(CustomPropTypes.mineName),
+  minespaceUserEmailHash: PropTypes.objectOf(PropTypes.any),
+  refreshData: PropTypes.func,
+};
+
+const defaultProps = {
+  mines: [],
+  minespaceUserEmailHash: {},
+  refreshData: () => {},
+};
```

Comment:
This doesn't appear to be getting used in this file: 'refreshData', did you have linting errors? It can be removed if not used, and removed from the 'OpenModal' function